src/patternfly/components/Wizard/wizard.scss

```diff
@@ -320,6 +329,12 @@
   line-height: var(--pf-c-wizard__toggle-icon--LineHeight);
 }
 
+.pf-c-wizard__drawer-toggle {
+  display: var(--pf-c-wizard__drawer-toggle--Display);
+  float: right;
```

Comment:
🥸

Stuff with text is going to flow around it like this - is that intended?

<img width="612" alt="Screen Shot 2022-06-15 at 7 20 42 PM" src="https://user-images.githubusercontent.com/35148959/173963602-37c72048-1a44-40f4-9127-067938825b7e.png">


<img width="1174" alt="Screen Shot 2022-06-15 at 7 21 03 PM" src="https://user-images.githubusercontent.com/35148959/173963597-e3d5ab2b-9878-46a4-9c01-0f82e5d8776f.png">

Reply:
Hmm that's a good point @mcoker - i'd probably at a minimum make sure that the header doesn't wrap around the button like that (or whatever is on the same line as the button). Whatever is below though, like the description, I think is ok to have wrap that way!